src/content/tutorials/user-auth-with-auth0/_index.md

```diff
@@ -0,0 +1,736 @@
+---
+title: 'User Auth with Auth0'
+showNew: true
+---
+
+In this tutorial, you'll integrate Auth0, an authentication and authorization provider, into a Cloudflare Workers application. Adding authorization and authentication to an application is a common task for developers, and by implementing it using Cloudflare Workers, you can take advantage of Workers' unique platform advantages to simplify how and when your application needs user data.
+
+## What you'll learn
+
+- How to authorize and authenticate users in Workers
+- How to persist authorization credentials inside of Workers KV
+- How to use Auth0 user info inside of your Workers application
+
+## Prerequisites
+
+To publish your Worker to Cloudflare, you'll need a few things:
+
+- A Cloudflare account, and access to the API keys for that account
+- An Auth0 account
+- A Wrangler installation running locally on your machine, and access to the command-line
+
+If you don't have those things quite yet, don't worry. We'll walk through each of them and make sure we're ready to go, before you start creating your application.
```

Comment:
You use 2nd person singular "you" here, but the rest of the tutorial uses plural "we"